docs/contracts.rst

```diff
@@ -1145,6 +1145,8 @@ Example of a Function Type (a variable declaration, where the variable is of typ
 
 Abstract contracts decouple the definition of a contract from its implementation providing better extensibility and self-documentation and
 facilitating patterns like the `Template method <https://en.wikipedia.org/wiki/Template_method_pattern>`_ and removing code duplication.
+Abstract contracts are useful in the same way that defining methods in an Interface is useful. It's a way for the designer of the Abstract contract to say "any child of mine MUST implement this method".
```

Comment:
Please do not needlessly capitalize words, we use `abstract` and `interface` just like regular words. Also please do not use contractions.

Reply:
`It's` - this is written and not spoken text.

Reply:
@chriseth Thanks. Fixed both issues.